data/apis/Amberdata/deployments/2022-01-31/airkeeper.json

```diff
@@ -0,0 +1,1016 @@
+{
+  "chains": [
+    {
+      "contracts": {
+        "RrpBeaconServer": "0x2cFda716b751eb406C5124C6E4428F2AEA453D96"
+      }
+    },
+    {
+      "contracts": {
+        "RrpBeaconServer": "0x2cFda716b751eb406C5124C6E4428F2AEA453D96"
+      }
+    },
+    {
+      "contracts": {
+        "RrpBeaconServer": "0x2cFda716b751eb406C5124C6E4428F2AEA453D96"
+      }
+    },
+    {
+      "contracts": {
+        "RrpBeaconServer": "0xFF298a3169117606BDA78eD6C04917224968f9b5"
+      }
+    }
+  ],
+  "triggers": {
+    "rrpBeaconServerKeeperJobs": [
+      {
+        "templateId": "0x2a0219013e042c3d68afa6501e9798c84863e7e466f64f4a7d45cb00bf70732e",
+        "templateParameters": [
+          {
+            "name": "pair",
+            "type": "bytes32",
+            "value": "avax_usd"
+          },
```

Comment:
Just wondering why not putting this param in the `overrideParameters` and create a single template that can be reused on all other jobs since _path, _type, _times and loopbackPeriod seems to be the same 🤔 This way we would only have to create a single template tx.

Reply:
This is the file from the deployment as-is, so I wouldn't change it here.
The purpose of templates AFAIK, is to save on gas; there's a trade-off between deploying lots of templates (initial high gas cost) vs deploying one template and overriding parameters. This is akin to normalisation in databases. We've essentially de-normalised the data here to reduce future gas costs.